src/test/java/uk/gov/pay/api/it/fixtures/PaymentSearchResultBuilder.java

```diff
@@ -21,6 +21,8 @@
     private static final int DEFAULT_NUMBER_OF_RESULTS = 3;
 
     public static final String DEFAULT_CREATED_DATE = DateTimeUtils.toUTCDateString(ZonedDateTime.now());
+    public static final String DEFAULT_CAPTURE_SUBMIT_TIME = DateTimeUtils.toUTCDateString(ZonedDateTime.now());
```

Comment:
Yes, one of the requirements is that everything should be UTC - and that's what's coming from connector, so the conversion should not actually change the time zone.